jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -36,35 +36,26 @@ see the [Configuring CircleCI]({{ site.baseurl }}/2.0/configuration-reference/#p
 
 ## Using the CircleCI CLI to Split Tests
 
-CircleCI supports automatic test allocation
-across your containers.
-The allocation is file-based.
-It requires the CircleCI CLI,
-which is automatically injected
-into your build at run-time.
+CircleCI supports automatic test allocation across your containers. The allocation is file-based. It requires the CircleCI CLI, which is automatically injected into your build at run-time.
 
-To install the CLI locally,
-see the [Using the CircleCI Local CLI]({{ site.baseurl }}/2.0/local-cli/) document.
+To install the CLI locally, see the [Using the CircleCI Local CLI]({{ site.baseurl }}/2.0/local-cli/) document.
 
 ### Globbing Test Files
```

Comment:
If folks are using the timing-data split method, do they still need this globbing info? Do they still do the globbing to show which test files they want to use?

Reply:
`circleci tests glob` is provided as a convenience to create the list of test thingies (in this case files) that are to be passed to `circleci tests split`. It's entirely optional whether you use it or not.

E.g. `bundle exec rspec $(circleci tests glob test/**/*_test.rb | circleci tests split)`

Reply:
Thanks!